src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -48,57 +42,72 @@ const renderLink = ({ cssClasses, createURL, refine, templateProps }) => (
   );
 };
 
-class Breadcrumb extends PureComponent {
-  static propTypes = {
-    createURL: PropTypes.func,
-    cssClasses: PropTypes.objectOf(PropTypes.string),
-    items: itemsPropType,
-    refine: PropTypes.func.isRequired,
-    separator: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
-    templateProps: PropTypes.object.isRequired,
-    translate: PropTypes.func,
-  };
-
-  render() {
-    const { createURL, items, refine, cssClasses } = this.props;
-
-    const breadcrumb = items.map(renderLink(this.props));
-
-    const homeClassnames = cx(cssClasses.item, {
-      [cssClasses.selectedItem]: items.length === 0,
-    });
+const Breadcrumb = ({
```

Comment:
i'm curious on why we don't choose not to adopt more PureComponent?
Any reason for this @algolia/instantsearch-for-websites

Reply:
It hasn't proved necessary / useful to use PureComponent when we tried it the previous time. Note that this is Preact, not React, so they don't behave exactly the same

Reply:
Inside the connector we often times create new functions (e.g. refine) on each render so the `PureComponent` check would fail every time. It means that we will do the check for nothing (not in that case though). But as @Haroenv said until we have performance issues then we can stick with functional component.